### DISCUSSION TITLE
关于是否有必要优先使用 OpenJDK 启动游戏替代优先使用 Oracle JDK 启动游戏

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#3952] [提问] [answered]
如题，Oracle JDK 优化真的很差，同配置下 OpenJDK 卡顿现象明显比 Oracle JDK少

**Comment by allMagicNB** [comment]
你猜猜 PCL 有优先选择 Oracle JDK 吗？

**Reply by wuliaodexiaoluo** [reply]
我 N 个 Java
甚至有个 OpenJDK 版本比 Oracle JDK 还高，PCL 自动选择还是优先 Oracle JDK

**Comment by hejiehao** [comment]
这应当没有明显的性能差异

**Comment by LTCatt** [comment, marked as answer]
PCL 不会内置优先某类 JDK 的策略，有需要可以自己折腾
